src/lib/cmdk/components/Command.svelte

```diff
@@ -86,13 +86,13 @@
 </script>
 
 {#if asChild}
-	<slot {root} label={{ attrs: labelAttrs }} />
```

Comment:
We should move these into a reactive `slotProps` prop, so something like:

```svelte
<script lang="ts">
	$: slotProps = {
		root,
		label: { attrs: labelAttrs },
		stateStore,
		state: $stateStore,
	}
</script>
```

If we're going to pass stores as slot props I think they should be suffixed with `Store` and then we can also pass the value of the store as the `state` prop.